Testing/VelaptorTests/Content/ContentPathResolverTests.cs

```diff
@@ -121,11 +122,12 @@ public void ResolveFilePath_WhenContentNameEndsWithDirSeparator_ThrowsException(
         // Arrange
         var resolver = new ContentPathResolverFake();
 
-        // Act & Assert
-        AssertExtensions.ThrowsWithMessage<ArgumentException>(() =>
-        {
-            resolver.ResolveFilePath($@"{contentName}");
-        }, $@"The '{contentName}' cannot end with a folder.  It must end with a file name with or without the extension. (Parameter 'contentName')");
+        // Act
+        var act = () => resolver.ResolveFilePath($@"{contentName}");
```

Comment:
Would you be able to please refactor this to what is shown below?
I figured we might as well since we have the PR going.  :)

```suggestion
        var act = () => resolver.ResolveFilePath(contentName);
```